R/RLearner_regr_gpfit.R

```diff
@@ -0,0 +1,61 @@
+#' @export
+makeRLearner.regr.gpfit = function(){
+  makeRLearnerRegr(
+    cl = "regr.gpfit",
+    package = "GPfit",
+    par.set = makeParamSet(
+      makeNumericVectorLearnerParam(id = "control", len = 3, lower = c(1, 1, 1)),
+      makeNumericLearnerParam(id = "nug_thres", default = 20, lower = 10, upper = 25),
+      makeLogicalLearnerParam(id = "trace", default = FALSE, tunable = FALSE),
+      makeIntegerLearnerParam(id = "maxit", default = 100, lower = 0),
+      makeUntypedLearnerParam(id = "optim_start", tunable = FALSE),
+      makeLogicalLearnerParam(id = "scale", default = TRUE)
```

Comment:
zu weiteren erklärung, auch wenn hier der falsche platz für diese diskussionen ist:
es ist ja (vermutlich) richtig, dass du alles mit jakob absprichst, denn ich denke mal, dass du sein hiwi bist. was heißt dass er (vermutlich) bestimmen kann was du so alles tun musst.

was in die software gemergt wird, bestimmt aber eine gruppe von leuten. wir versuchen das hier halbwegs im konsens mit einigen "älteren" leuten zu machen, indem wir uns gegenseitig helfen, durch reviews. am ende zählt vorallem das was "richtig" ist. (so wie hier)

und manchmal bestimme ich oder ein anderer der core devels (was hoffenltich sehr selten notwendig ist, weil klar sein sollte was richtig ist)